.pre-commit-config.yaml

```diff
@@ -1,39 +1,35 @@
 default_language_version:
-    python: python3.6
+  python: python3.6
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v3.4.0
     hooks:
-    -   id: check-yaml
-    -   id: check-json
-    -   id: check-added-large-files
-    -   id: end-of-file-fixer
-    -   id: trailing-whitespace
-    -   id: check-case-conflict
-    -   id: mixed-line-ending
+      - id: check-yaml
+      - id: check-json
+      - id: check-added-large-files
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: check-case-conflict
+      - id: mixed-line-ending
 
--   repo: https://github.com/psf/black
+  - repo: https://github.com/psf/black
     rev: 20.8b1
     hooks:
-    -   id: black
+      - id: black
 
--   repo: https://github.com/timothycrosley/isort
+  - repo: https://github.com/timothycrosley/isort
     rev: 5.7.0
     hooks:
-    -   id: isort
+      - id: isort
 
--   repo: https://gitlab.com/pycqa/flake8
+  - repo: https://gitlab.com/pycqa/flake8
     rev: 3.8.4
     hooks:
-    -   id: flake8
+      - id: flake8
         additional_dependencies: [flake8-bugbear]
 
-# -   repo: https://github.com/nbQA-dev/nbQA
-#     rev: 0.5.7
-#     hooks:
-#     -   id: nbqa-black
-#         additional_dependencies: [black==20.8b1]
-#     -   id: nbqa-isort
-#         additional_dependencies: [isort==5.7.0]
-#     -   id: nbqa-flake8
-#         additional_dependencies: [flake8==3.8.4]
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.2.1
+    hooks:
+      - id: prettier
+        args: [--print-width=90, --prose-wrap=always]
```

Comment:
Args like these is what I find annoying.

I don't see the point of being so strict about line length with markdown files (since I assume everyone has line wrapping set in their editor/IDE for Markdown anyway). And the `prose-wrap=always` leads to messier diffs.